### ISSUE TITLE
Pseudoglioma

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9590]
**Preferred term label:**
Pseudoglioma

**Synonyms**

**Definition (free text, please give PubMed ID)**
Hyperplasia of the vitreous, and eye and corneal opacities that mimic an intraocular glioma.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 259770 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000262
- label: Pseudoglioma
- parents: [Abnormal vitreous humor morphology (HP:0004327)](https://hpo.jax.org/app/browse/term/HP:0004327)
- pmid: PMID:16252235
- def: A circumscribed whitish-yellow mass in the posterior chamber associated with retinal folds and sometimes retinal detachment and is said to resemble a retinal glioma.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9590
- synonyms:
